src/sharedHelpers/fetchPlaceName.js

```diff
@@ -32,15 +33,12 @@ const setPlaceName = ( results: Array<Object> ) => {
   return placeName;
 };
 
-const fetchPlaceName = async ( lat: number, lng: number ): Promise<?string> => {
-  try {
-    const results = await Geocoder.geocodePosition( { lat, lng } );
-    if ( results.length === 0 ) { return null; }
-    return setPlaceName( results );
-  } catch ( e ) {
-    console.warn( e, "couldn't fetch geocoded position with coordinates: ", lat, lng );
-    return null;
-  }
+const fetchPlaceName = async ( lat: ?number, lng: ?number ): any => {
+  const { isConnected } = await NetInfo.fetch( );
```

Comment:
I think you want `isInternetReachable` here. The problem is that we need to know if we can connect to Google, which requires a network connection (`isConnected`), but that network has to be connected to the Internet (`isInternetReachable`).